charts/fluentd-elasticsearch/values.yaml

```diff
@@ -54,6 +54,7 @@ elasticsearch:
   host: "elasticsearch-client"
   logstashPrefix: "logstash"
   port: 9200
+  path: ""
```

Comment:
have you testet, that the adding of en empty path value does not affect users which are updating from an older chart version?

Reply:
The default behavior here is to set the default value to "", which is the default value in the ES plugin. If a user is upgrading from an older chart version, they would use the default path value of ""